Missing icon path is causing main build to break

## Description

https://github.com/mozilla-sre-deploy/deploy-bedrock/actions/runs/11278649114/job/31367588781

```
 #48 4.928 Post-processing 'css/m24-navigation-and-footer.css' failed!
#48 4.928 
#48 4.929 Traceback (most recent call last):
#48 4.929   File "/app/manage.py", line 14, in <module>
#48 4.929     execute_from_command_line(sys.argv)
#48 4.930   File "/venv/lib/python3.12/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
#48 4.930     utility.execute()
#48 4.930   File "/venv/lib/python3.12/site-packages/django/core/management/__init__.py", line 436, in execute
#48 4.930     self.fetch_command(subcommand).run_from_argv(self.argv)
#48 4.930   File "/venv/lib/python3.12/site-packages/django/core/management/base.py", line 412, in run_from_argv
#48 4.930     self.execute(*args, **cmd_options)
#48 4.930   File "/venv/lib/python3.12/site-packages/django/core/management/base.py", line 458, in execute
#48 4.931     output = self.handle(*args, **options)
#48 4.931              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#48 4.931   File "/venv/lib/python3.12/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 209, in handle
#48 4.931     collected = self.collect()
#48 4.931                 ^^^^^^^^^^^^^^
#48 4.932   File "/venv/lib/python3.12/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 154, in collect
#48 4.932     raise processed
#48 4.932   File "/venv/lib/python3.12/site-packages/django/contrib/staticfiles/storage.py", line 372, in _post_process
#48 4.932     content = pattern.sub(converter, content)
#48 4.932               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#48 4.932   File "/venv/lib/python3.12/site-packages/django/contrib/staticfiles/storage.py", line 249, in converter
#48 4.932     hashed_url = self._url(
#48 4.932                  ^^^^^^^^^^
#48 4.932   File "/venv/lib/python3.12/site-packages/django/contrib/staticfiles/storage.py", line 182, in _url
#48 4.933     hashed_name = hashed_name_func(*args)
#48 4.933                   ^^^^^^^^^^^^^^^^^^^^^^^
#48 4.933   File "/venv/lib/python3.12/site-packages/django/contrib/staticfiles/storage.py", line 422, in _stored_name
#48 4.933     cache_name = self.clean_name(self.hashed_name(name))
#48 4.933                                  ^^^^^^^^^^^^^^^^^^^^^^
#48 4.933   File "/venv/lib/python3.12/site-packages/django/contrib/staticfiles/storage.py", line 143, in hashed_name
#48 4.934     raise ValueError(
#48 4.934 ValueError: The file 'img/icons/close.svg' could not be found with <django.contrib.staticfiles.storage.ManifestStaticFilesStorage object at 0x7f3281ee6840>.
#48 ERROR: process "/bin/sh -c honcho run --env docker/envfiles/prod.env docker/bin/build_staticfiles.sh" did not complete successfully: exit code: 1
failed to solve: process "/bin/sh -c honcho run --env docker/envfiles/prod.env docker/bin/build_staticfiles.sh" did not complete successfully: exit code: 1
------
 > [release release 3/5] RUN honcho run --env docker/envfiles/prod.env docker/bin/build_staticfiles.sh:
4.932                  ^^^^^^^^^^
make: *** [Makefile:158: build-ci] Error 17
4.932   File "/venv/lib/python3.12/site-packages/django/contrib/staticfiles/storage.py", line 182, in _url
4.933     hashed_name = hashed_name_func(*args)
4.933                   ^^^^^^^^^^^^^^^^^^^^^^^
4.933   File "/venv/lib/python3.12/site-packages/django/contrib/staticfiles/storage.py", line 422, in _stored_name
4.933     cache_name = self.clean_name(self.hashed_name(name))
4.933                                  ^^^^^^^^^^^^^^^^^^^^^^
4.933   File "/venv/lib/python3.12/site-packages/django/contrib/staticfiles/storage.py", line 143, in hashed_name
4.934     raise ValueError(
4.934 ValueError: The file 'img/icons/close.svg' could not be found with <django.contrib.staticfiles.storage.ManifestStaticFilesStorage object at 0x7f3281ee6840>.
------
Error: Process completed with exit code 2.
```

--- a/media/css/m24/components/navigation-refresh.scss
+++ b/media/css/m24/components/navigation-refresh.scss
@@ -81,14 +81,13 @@
     }
 
     &.mzp-is-active {
-        background-color: $color-marketing-gray-20;
-        background-image: url('#{$image-path}/icons/close.svg');
-    }
+        background: url('/media/protocol/img/icons/close.svg') center center no-repeat;
+        width: 40px;
+        @include image-replaced;
 
-    &:not(.mzp-is-active) {
-        text-indent: unset;
-        background-image: none;
-        width: fit-content;
+        &::after {
+            content: none;
+        }
     }
 
     @media #{$mq-md} {

